p/bvg/index.js

```diff
@@ -19,8 +19,8 @@ import {products} from './products.js'
 // todo: there's also a referenced icon `{"res":"occup_fig_{low,mid}"}`
 const addOccupancy = (item, occupancyCodes) => {
 	const remIdx = (item.remarks || [])
-	.findIndex(r => r.code && occupancyCodes.has(r.code))
-	if (remIdx < 0) return;
+		.findIndex(r => r.code && occupancyCodes.has(r.code))
+	if (remIdx < 0) return
```

Comment:
An aspect that I forgot to mention: I like to put `;` behind `return` so that, if the next line starts with an expression (e.g. `foo.bar()`), I don't accidentally return the result of the expressions, but `void`.

Is it hard to add this rule?

Reply:
The options for setting semicolons are "always" (default) and "never" (the current setting). I don't understand overriding rules well enough to say whether it is possible to create an exception for result with acceptable effort.
We could also disable the rule, but then it becomes inconsistent again.

We could also switch off the rule completely.

Reply:
I don't mind either way, pick one. I'm used to omitting semicolons, as they're *slightly* less visual noise, but they really help prevent bugs as explained above, so it's a tie for me.

Reply:
Okay, I see the point of visual noise, but prevent bugs is more significant. I also like to use the defaults and in this case semicolons are set by default. I just made a commit that adjusts the rule. Of course, this affects a huge amount of lines in the code.